xcodeproj/internal/bazel_integration_files/import_indexstores.sh

```diff
@@ -101,7 +101,7 @@ remaps=(
   # tools based paths as well.
   # `DEVELOPER_DIR` has an optional `./` prefix, because index-import adds `./`
   # to all relative paths.
-  -remap "(?:.*?/[^/]+/Contents/Developer|(?:./)?DEVELOPER_DIR|/Library/Developer/CommandLineTools).*?/SDKs/([^\\d.]+)=$DEVELOPER_DIR/Platforms/\$1.platform/Developer/SDKs/\$1"
+  -remap "^(?:.*?/[^/]+/Contents/Developer|(?:./)?DEVELOPER_DIR|/PLACEHOLDER_DEVELOPER_DIR|/Library/Developer/CommandLineTools).*?/SDKs/([^\\d.]+)=$DEVELOPER_DIR/Platforms/\$1.platform/Developer/SDKs/\$1"
```

Comment:
I think the `^` prefix here is safe, and theoretically makes the regex more efficient, but I can leave it out if you'd prefer

Reply:
Let’s leave it off for this PR, and add it in another PR.